utils/generate-kubeconfig-file.sh

```diff
@@ -0,0 +1,56 @@
+#!/usr/bin/env sh
+
+SCRIPTDIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd )"
+
+tmpDir=$(mktemp -d)
+
+function error() {
+    echo "$@" 1>&2
+}
+if [ -z "$OFFLINE_TOKEN_FILE" ]; then
+  error "Error: Environment variable OFFLINE_TOKEN_FILE not specified"
+  error ""
+  error "- Obtain one by navigating to https://console.redhat.com/openshift/token"
+  error "- copy token and save to a file"
+  error "- export the path to the file in an environment variable"
+  error "  % export OFFLINE_TOKEN_FILE=<location>"
+  error ""
+  exit 1
+fi
+
+if [ ! -f "$OFFLINE_TOKEN_FILE" ]; then
+  error "Error: OFFLINE_TOKEN_FILE: ${OFFLINE_TOKEN_FILE} does not exist"
+  exit 1
+fi
+
+if [ -z "${WORKSPACE}" ]; then
+  error "Error: Environment variable WORKSPACE not specified"
+  error ""
+  error "- This should contain the name of the development/user workspace"
+  error "- It should NOT contain the suffix - '-tenant'"
```

Comment:
Why do we do things like this? It is weird to me to pass partial namespace names as arguments

Reply:
Got it, thanks